app/views/devise/registrations/_edit_form.html.erb

```diff
@@ -0,0 +1,71 @@
+<%= form_for(resource, as: resource_name,
+             url: registration_path(resource_name), html: { method: :put }) do |f| %>
+
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="row">
+    <div class="col-12 col-md-6">
+      <div class="form-group">
+        <%= f.label :first_name %><br/>
+        <%= f.text_field :first_name, autofocus: true, class: 'form-control' %>
+      </div>
+    </div>
+    <div class="col-12 col-md-6">
+      <div class="form-group">
+        <%= f.label :last_name %><br/>
+        <%= f.text_field :last_name, class: 'form-control' %>
+      </div>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-12 col-md-6">
+      <%= f.label :email %>
+      <div class="form-group">
+        <%= f.email_field :email, autocomplete: 'email', class: 'form-control' %>
+      </div>
+
+      <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
+        <div><%= t('waiting_confirmation_for') %>: <%= resource.unconfirmed_email %></div>
+      <% end %>
+    </div>
+    <div class="col-12 col-md-6">
+      <div class="form-group">
+        <%= f.label :current_password %>
+        <%= f.password_field :current_password, autocomplete: 'current-password', class: 'form-control' %>
+      </div>
+    </div>
+  </div>
+  <div class="row">
+    <a class="btn btn-link"
+       data-toggle="collapse"
+       href="#newPassword"
+       role="button"
+       aria-expanded="false"
+       aria-controls="newPassword">
+      Set a new password
+    </a>
+  </div>
+  <div class="collapse" id="newPassword">
+    <% if @minimum_password_length %>
+      <em>(<%= "#{@minimum_password_length} #{t('min_char')}" %>)</em>
+    <% end %>
+    <div class="row">
+      <div class="col-12 col-md-6">
+        <div class="form-group">
+          <%= f.label :password %> <br/>
+          <%= f.password_field :password, autocomplete: "new-password", class: 'form-control' %>
+        </div>
+      </div>
+      <div class="col-12 col-md-6">
+        <div class="form-group">
+          <%= f.label :password_confirmation %><br/>
+          <%= f.password_field :password_confirmation, autocomplete: 'new-password', class: 'form-control' %>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="actions">
```

Comment:
Lets keep the class name singular as use two word for class name. May be `registration-action` or any other better name?
![Screen Shot 2021-06-21 at 12 14 51 PM](https://user-images.githubusercontent.com/6076762/122710392-59de6900-d28a-11eb-9b69-bbfc8a542a3a.png)
![Screen Shot 2021-06-21 at 12 14 29 PM](https://user-images.githubusercontent.com/6076762/122710398-5c40c300-d28a-11eb-841f-82b6338a628d.png)

Reply:
Nice catch! It's actually a totally useless div, I removed it. It's an unfortunate remains from Devise template. 
Thanks!